sphinx_gallery/tests/test_full.py

```diff
@@ -66,5 +66,15 @@ def test_embed_links(sphinx_app):
     assert 'scipy.signal.firwin.html' in lines
     assert '#module-numpy' in lines
     assert 'numpy.arange.html' in lines
-    # assert '#module-matplotlib.pyplot' in lines
-    # assert 'pyplot.html' in lines
+    assert '#module-matplotlib.pyplot' in lines
```

Comment:
Intersesting, I remember this was failing from time to time, but maybe now this is more stable?

Reply:
yeah the CIs are happy so I figure we can try it, if it ends up being inconsistent again we can comment it back out